src/cinder/app/linux/WindowImplLinuxGlfw.cpp

```diff
@@ -74,7 +75,12 @@ WindowImplLinux::WindowImplLinux( const Window::Format &format, RendererRef shar
     ::glfwWindowHint( GLFW_SAMPLES, options.getMsaa() );
 
 	auto windowSize = format.getSize();
-	mGlfwWindow = ::glfwCreateWindow( windowSize.x, windowSize.y, format.getTitle().c_str(), NULL, NULL );
+	if( mFullScreen ) {
+		mGlfwWindow = ::glfwCreateWindow( windowSize.x, windowSize.y, format.getTitle().c_str(), ::glfwGetPrimaryMonitor(), NULL );
```

Comment:
Is there a way to make this so it isn't always the primary monitor, but the monitor / `ci::Display` that is associated with this `Window`?